web/static/js/react/nav-bar.js

```diff
@@ -0,0 +1,64 @@
+var NavBar = React.createClass({
+  // Props are attributes we pass into this React component
+  propTypes: {
+    // isLoggedIn tells us whether the user is logged in
+    isLoggedIn: React.PropTypes.bool.isRequired
+  },
+
+  // opens the menu
+  openMenu() {
+    console.log("opening menu");
+  },
+
+  renderDashboardLink() {
+    if(this.props.isLoggedIn) {
+      return(
+        <li>
+          <a href="/#/dashboard">Dashboard</a>
+        </li>
+      );
+    }
+  },
```

Comment:
These functions show how you can decide whether to show HTML based on a prop passed in. Here we have the user not logged in so this li will never show, but if we change isLoggedIn to true it would show this li and not the login and signup li